consensus/src/rand/rand_gen/reliable_broadcast_state.rs

```diff
@@ -34,18 +38,23 @@ impl<D> AugDataCertBuilder<D> {
     }
 }
 
-impl<S: Share, D: AugmentedData> BroadcastStatus<RandMessage<S, D>, RandMessage<S, D>>
+impl<S: TShare, D: TAugmentedData> BroadcastStatus<RandMessage<S, D>, RandMessage<S, D>>
     for Arc<AugDataCertBuilder<D>>
 {
     type Aggregated = CertifiedAugData<D>;
     type Message = AugData<D>;
     type Response = AugDataSignature;
 
     fn add(&self, peer: Author, ack: Self::Response) -> anyhow::Result<Option<Self::Aggregated>> {
+        debug!(
```

Comment:
We may want to clean up these logs later..

Reply:
fixed

Reply:
fixed